src/content/home.js

```diff
@@ -1,4 +1,14 @@
 const home = {
+  introduction: {
+    title: 'AnitaB.org Open Source',
+    image: {
+      source: require('./../assets/landing/landing.png'),
+    },
+    content: {
+      par:
+        'Lorem ipsum dolor sit amet consectetur, adipisicing elit. Quis omnis minus, quisquam ab architecto dolor hic dolores, ducimus explicabo, voluptates fugit dolorum itaque enim nam. Enim nulla, vitae recusandae est pariatur consequuntur aperiam molestiae, minus vero, fugiat ipsam facilis cupiditate alias dolorem mollitia officiis. Excepturi, blanditiis?',
```

Comment:
Why random text is used here? Is it intentional?

Reply:
Thanks for pointing this out. The real content is not there yet, so I used random text. I've also updated the description to inform this.

Reply:
no `Lorem ipsum` will be merged. Rise issue and wait for content or write content by your own. Thanks!